plugins/quickinspector/quickitemmodel.cpp

```diff
@@ -156,7 +155,14 @@ void QuickItemModel::connectItem(QQuickItem *item)
 
 void QuickItemModel::disconnectItem(QQuickItem *item)
 {
-    disconnect(item, nullptr, this, nullptr);
+    disconnect(item, SIGNAL(parentChanged(QQuickItem*)), this, SLOT(itemReparented()));
+    disconnect(item, SIGNAL(visibleChanged()), this, SLOT(itemUpdated()));
+    disconnect(item, SIGNAL(focusChanged(bool)), this, SLOT(itemUpdated()));
+    disconnect(item, SIGNAL(activeFocusChanged(bool)), this, SLOT(itemUpdated()));
+    disconnect(item, SIGNAL(widthChanged()), this, SLOT(itemUpdated()));
+    disconnect(item, SIGNAL(heightChanged()), this, SLOT(itemUpdated()));
+    disconnect(item, SIGNAL(xChanged()), this, SLOT(itemUpdated()));
+    disconnect(item, SIGNAL(yChanged()), this, SLOT(itemUpdated()));
```

Comment:
connectItem() is creating a new monitor each time it handle an item, but this does not delete it, resulting to possible multiple filters on the item.
Maybe we want to uninstall and delete the previous event filter here ?
Also the created monitor does have this as parent meaning the monitor will not be deleted until this is deleted, this let the monitor in memory when item is deleted for nothing.